### ISSUE TITLE
Weekly Newsletter Issue - send to outside email doesn't persist

### ISSUE THREAD

**Issue by Nvious1** [tidusjar/PlexRequests.Net#643]
If this is a bug report please make sure you have filled the following in:
(If it's not a bug and a feature request then just remove the below)

#### Plex Requests.Net Version:
1.9.6

#### Operating System:
Server 2012 R2

#### Reproduction Steps:

In the newsletter settings, I Select Enable Newsletter, but then have Send to all Plex users unchecked. Then I added email1@blah.com;email2@blah.com and hit submit.  A notice states that it has been successfully save.  Once I leave the settings page and come back the email addresses are no longer persisting.  No logs are through even with debug enable.  I change the cron schedule for the email to run and confirm no email was delivered.

I would like to basically be able to only send the newsletters to folks that want that information.  These users are part of my friends list however not all of them want the notificaiton so it would be ideal if you could be select as to "what friends" you send to.

**Comment by tidusjar** [comment]
I'll fix the bug, the last part is going to be part of the user management
stuff

**Comment by Nvious1** [comment]
You all are awesome.  Please keep up the great work!

**Comment by tidusjar** [comment]
Here is the fixed version that I will release if it fixes your issue: https://ci.appveyor.com/project/tidusjar/requestplex/build/1.9.921/artifacts

Can you test this?

**Comment by Nvious1** [comment]
Sorry for the delay.  Yes this fixed it.

**Comment by Nvious1** [comment]
After my cron fired this morning to send the email, I saw these in the error log.  I did receive the email correctly.

Exception when trying to process a Movie, either in getting the metadata from Plex OR getting the information from TheMovieDB, Plex GUID = com.plexapp.agents.imdb://tt4255252?lang=en	PlexRequests.Services.Jobs.RecentlyAdded	Error	

11/3/2016 7:37:04 AM
System.ArgumentNullException: Value cannot be null. Parameter name: source at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source) at PlexRequests.Services.Jobs.RecentlyAdded.GenerateMovieHtml(RecentlyAddedModel movies, PlexSettings plexSettings, StringBuilder sb) in C:\projects\requestplex\PlexRequests.Services\Jobs\RecentlyAdded.cs:line 154	PlexRequests.Services.Jobs.RecentlyAdded	Error	

11/3/2016 7:37:02 AM
System.NullReferenceException: Object reference not set to an instance of an object. at PlexRequests.Helpers.StringHelper.d__3.MoveNext() in C:\projects\requestplex\PlexRequests.Helpers\StringHelper.cs:line 81 at PlexRequests.Services.Jobs.RecentlyAdded.Send(NewletterSettings newletterSettings, String html, PlexSettings plexSettings, Boolean testEmail) in C:\projects\requestplex\PlexRequests.Services\Jobs\RecentlyAdded.cs:line 268 at PlexRequests.Services.Jobs.RecentlyAdded.Start(NewletterSettings newletterSettings, Boolean testEmail) in C:\projects\requestplex\PlexRequests.Services\Jobs\RecentlyAdded.cs:line 125 at PlexRequests.Services.Jobs.RecentlyAdded.Execute(IJobExecutionContext context) in C:\projects\requestplex\PlexRequests.Services\Jobs\RecentlyAdded.cs:line 89	PlexRequests.Services.Jobs.RecentlyAdded	Error
